Solution

meryam99 opened the pull request:
https://meryam99.github.io/react_list-of-goods-js/

meryam99 committed:
Solution

Moroz-Dmytro commented on src/App.jsx:
```diff
@@ -13,34 +15,91 @@ export const goodsFromServer = [
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversedField, setReversedField] = useState(false);
```
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```

Moroz-Dmytro replied:
Not fixed

Moroz-Dmytro commented on src/App.jsx:
```diff
@@ -13,34 +15,91 @@ export const goodsFromServer = [
+    sortField,
+    reversedField,
+  });
+  const isChangedOrder = sortField !== '' || reversedField;
```
```suggestion
  const isResetButtonVisible = !!sortField || reversedField;
```

Moroz-Dmytro replied:
Not fixed

Moroz-Dmytro commented on src/trash.jsx:
Delete this file

Moroz-Dmytro reviewed (changes requested):
Please, fix your DEMO link, it's broken 😿

meryam99 committed:
Solution

meryam99 commented:
> Please, fix your DEMO link, it's broken 😿

Hello,
Deleted the 'trash' file (sorry, totally forgot about the draft :(
Fixed the demo-link

Thanks!

Moroz-Dmytro reviewed (changes requested):
Some comments are not fixed 🤔

meryam99 committed:
Solution

meryam99 commented:
> Some comments are not fixed 🤔

Hello and sooo sorry
Didn't see for the first time

meryam99 committed:
Solution

Moroz-Dmytro reviewed (approved):
LGTM 🚀
